Source/VMware.vSphereDSC/Tests/Integration/VMHostVss.Integration.Tests.ps1

```diff
@@ -36,12 +36,12 @@ $script:dscResourceName = 'VMHostVss'
 $script:dscConfig = $null
 $script:moduleFolderPath = (Get-Module VMware.vSphereDSC -ListAvailable).ModuleBase
 $script:integrationTestsFolderPath = Join-Path (Join-Path (Join-Path $moduleFolderPath 'Tests') 'Integration') $script:dscResourceName
-$script:configurationFile = "$script:integrationTestsFolderPath\Configurations\$($script:dscResourceName)\$($script:dscResourceName)_Config.ps1"
 
 $script:configWithNewVss = "$($script:dscResourceName)_New_Config"
 $script:configWithModifyVss = "$($script:dscResourceName)_Modify_Config"
 $script:configWithRemoveVss = "$($script:dscResourceName)_Remove_Config"
 
+$script:connection = Connect-VIServer -Server $Server -User $User -Password $Password
```

Comment:
The connection is not needed for this tests.

Reply:
Changed